docroot/sites/facilities.uiowa.edu/modules/facilities_core/facilities_core.module

```diff
@@ -239,9 +239,33 @@ function facilities_core_preprocess_field(&$variables) {
       // views_block:named_buildings-block_building using Views field api.
       if ($variables['element']['#view_mode'] === '_custom') {
         $variables['label_display'] = 'inline';
+        uiowa_core_set_attributes($variables, 'padding--inline-end--md');
       }
       break;
 
+    case 'field_building_lactation_rooms':
+    case 'field_building_rr_single_neutral':
+    case 'field_building_rr_single_men':
+    case 'field_building_rr_single_women':
+    case 'field_building_rr_multi_men':
+    case 'field_building_rr_multi_women':
+      // Wrap the field output in a <details> tag.
```

Comment:
This is fine, but seems like a fairly simple render array. https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Render%21Element%21Details.php/class/Details/10

Reply:
Great suggestion! Should be resolved in https://github.com/uiowa/uiowa/pull/7409/commits/b9346a14e195025d21753013314857334f66c61e.